src/sockets/matchmakingQueue.ts

```diff
@@ -65,16 +84,25 @@ export class MatchmakingQueue {
   }
 
   private checkQueue(): void {
-    if (this.queue.length < NUM_PLAYERS_PER_GAME) {
+    if (this.queue.length < MIN_PLAYERS_PER_GAME) {
       return;
     }
 
+    for (
```

Comment:
Can you add a comment saying why we want to go in reverse order. To prioritise larger games.